### DISCUSSION TITLE
Outbrain ads not blocked ?

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#3475] [Q&A]
Hello,
Today on igen.fr I saw some ads from Outbrain. I was surprised it was not blocked by default. 

I use 3 lists : Adguard DNS Filter, WindowSpyBlocker and oisd. 

Are ads getting smarter ?

![58DD2F39-5487-4B60-B8B1-4E05BFB5D9D9](https://user-images.githubusercontent.com/206181/130217094-a75306c4-bd24-4104-8748-c07ae46a122e.jpeg)

**Comment by ameshkov** [comment]
It probably should be reported to https://github.com/AdguardTeam/AdguardSDNSFilter, filters maintainers will see what can be done about this.

**Comment by ghost** [comment]
Hum, my iPhone doesn't block any ads. I checked on multiple site, in the Query Log I can see the iPhone IP but no blocking. Works perfectly on my computer. I use the DNS of Adguard directly on my router. 

Should I create an issue instead or continue here ?

**Comment by ghost** [comment]
Ok, it's my VPN.

If on my Phone I disable the VPN connection on the App (Windscribe), Adguard blocks ads, If I enable the VPN it doesn't work anymore. 

How it's possible ?

**Reply by ameshkov** [reply]
It's just the VPN decides what DNS server to use.
